### PR TITLE
Editorial updates for the Using Appsody section

### PR DESCRIPTION
**Pull request by SueChaplain**
- Ensure all references in text to Appsody are uppercase A
- Changes to some link titles
- Minor changes for layout.
- A few extra words here and there to help non-English speaking users follow the instructions.

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Related Issues

#issue-number

### CODE REVIEW

File: content/docs/using-appsody/building-and-deploying.md [hunk 1]
```diff
@@ -3,25 +3,29 @@ path: /docs/using-appsody/building-and-deploying
...
+These deployment options are covered in more detail in the following sections.
 
-If you want to generate a Docker image that matches what will be deployed on the target runtime platform, you can use the `appsody build` command.
+## Building your runtime Docker image with Appsody
 
-The `appsody build` command performs the following steps:
-1) Extracts your code and other artifacts that are necessary to build the deployment image from the development image, and saves all that in a directory located under `~/.appsody/extract`.
-2) Runs a `docker build` against a Dockerfile that was extracted on the previous step. This step produces an image in your local Docker registry. If you specified the `-t <tag>` parameter, the image will be tagged accordingly. If you ran `appsody build` with no parameters, the image will given a name that matches the name of your project.
+When you use the Appsody CLI to develop your applications, a *development* Docker image of the target runtime is downloaded and run for you. This image differs slightly from the image that is used at deployment time, because it configures tools that are useful only during the development phase.
```
Comment (mhamwala) [inline content/docs/using-appsody/building-and-deploying.md]: I think this is past tense here?
`downloaded and run for you` -> `downloaded and ran for you`

File: content/docs/using-appsody/building-and-deploying.md [hunk 2]
```diff
@@ -38,50 +42,62 @@ Built docker image appsody-project
...
-2) You must have configured your `kubectl` CLI to point to that cluster. The appsody CLI calls `kubectl` under the covers for you.
-3)  If you intend to push the Docker image containing your application to Docker Hub, your target cluster must be configured to be able to pull images from Docker Hub.
+Note: This deployment option is under development
+
+The `appsody deploy` command provides an example of how an application that you create with Appsody can be deployed directly to a Kubernetes cluster. The deployment occurs in the form of a Knative Serving service.
+
+To successfully run `appsody deploy` the following **pre-requisites** apply:
+
+- You must have access to a Kubernetes cluster, with Knative Serving installed and running. To install Knative locally by using the Kubernetes feature in Docker for Desktop, see [Installing Knative Locally](/docs/using-appsody/installing-knative-locally.md). To install Knative on other Kubernetes clusters, see the [Knative Install Documentation](https://knative.dev/docs/install/).
```
Comment (mhamwala) [inline content/docs/using-appsody/building-and-deploying.md]: possible reword:
`To install Knative locally by` - > `You can install Knative locally by..`

File: content/docs/using-appsody/building-and-deploying.md [hunk 3]
```diff
@@ -38,50 +42,62 @@ Built docker image appsody-project
...
+- You must configure your `kubectl` CLI to point to your Kubernetes cluster.
+- If you intend to push the Docker image containing your application to Docker Hub, your target cluster must be configured to pull images from Docker Hub.
 
 ### Deploying your application to a locally installed Kubernetes cluster
```
Comment (mhamwala) [inline content/docs/using-appsody/building-and-deploying.md]: We could possibly change this to?:
`Deploying your application to a local Kubernetes cluster`

Comment (mhamwala) [inline content/docs/using-appsody/building-and-deploying.md]: We could possibly change this to?:
`Deploying your application to a local Kubernetes cluster`

File: content/docs/using-appsody/building-and-deploying.md [hunk 4]
```diff
@@ -38,50 +42,62 @@ Built docker image appsody-project
...
-1) It calls `appsody build` and creates a deployment image, as described in the previous section.
-2) It tags the image with the special prefix `local.dev`, making it accessible to your Kubernetes cluster.
-3) It creates a Knative Serving manifest file named `appsody-service-nnnnnn.yaml`, where nnnnnn is a random numerical value. This yaml file is then used to issue a `kubectl apply -f` command against the target Kubernetes cluster. This yaml file is then used to issue a `kubectl apply -f` command against the target Kubernetes cluster.
+This command completes the following actions:
+
+- Calls `appsody build` and creates a *deployment* Docker image, as described in the previous section.
+- Tags the image with the special prefix `local.dev`, making it accessible to your Kubernetes cluster.
+- Creates a Knative Serving manifest file named `appsody-service-<nnnnnn>.yaml`, where `<nnnnnn>` is a random numerical value. This yaml file is used to issue a `kubectl apply -f` command against the target Kubernetes cluster.
 
 ### Deploying your application by pushing your image to Docker Hub
```
Comment (mhamwala) [inline content/docs/using-appsody/building-and-deploying.md]: Maybe change this to?:
`Deploying your application to Docker Hub`

File: content/docs/using-appsody/building-and-deploying.md [hunk 5]
```diff
@@ -38,50 +42,62 @@ Built docker image appsody-project
...
+- You must have developed a project with the appsody CLI, and you must have checked your project into a GitHub repository.
+- Your Kubernetes cluster must be configured to run Knative and Tekton pipelines (see [Installing Knative](https://knative.dev/docs/install/) and [Installing Tekton Pipelines](https://github.com/tektoncd/pipeline/blob/master/docs/install.md)).
+- Your project must include a Knative Serving deployment manifest called `appsody-service.yaml`, which is a current pre-requisite if you want to run the pipeline with minimal changes. A quick way to obtain such a manifest is to run the `appsody deploy` command described in the previous section. You need to rename the `appsody-service-<nnnnnn>.yaml` to `appsody-service.yaml`.
 
-More details on running the Tekton pipeline example for Appsody can be found in the [repo readme](https://github.com/appsody/tekton-example).
+More details on running the Tekton pipeline example for Appsody can be found in the repo [readme file](https://github.com/appsody/tekton-example).
```
Comment (mhamwala) [inline content/docs/using-appsody/building-and-deploying.md]: we could make this link explicit?:
[readme file] (https://github.com/appsody/tekton-example/blob/master/README.md)

File: content/docs/using-appsody/initializing-project.md [hunk 6]
```diff
@@ -29,18 +31,19 @@ java-spring-boot2	0.3.1   Spring Boot using IBM Java SDK and Maven
...
-Now you have a fully functional Appsody project. See the [Local Development](/docs/using-appsody/local-development.md) document for more details on running the project.
 
+## Configuring an existing project
 
-## Existing Project to Appsody
-You can configure an existing project to use Appsody, however you may have to manually adjust the project structure.
+You can configure an existing project to use Appsody, which typically involves adjusting the project structure.
 
-First, choose a stack that closely resembles your existing project. Use `appsody list` to view the available stacks, and review their documentation at <https://github.com/appsody/stacks/tree/master/incubator>.
+First, choose a stack that closely resembles your existing project. Use `appsody list` to view the available stacks, and review the [stack documentation](https://github.com/appsody/stacks/tree/master/incubator).
```
Comment (mhamwala) [inline content/docs/using-appsody/initializing-project.md]: we could change documentation to readme here:
..and review the appropriate [stack readme](https://github.com/appsody/stacks/tree/master/incubator).

File: content/docs/using-appsody/local-development.md [hunk 7]
```diff
@@ -3,26 +3,31 @@ path: /docs/using-appsody/local-development
...
+- run
+- debug
+- test
+
+Typically, you run one mode at a time. If you want to run more than one mode at a time you must specify unique name and port options to avoid conflicts between containers. See [Advanced Options](#advanced-options).
 
 While running the container in any of these modes, you can quit Appsody by pressing `Ctrl-C` or running `appsody stop` in the terminal.
 
-Each stack may open different ports and behave slightly differently across these modes. See your stack's documentation for further details.
+Each stack might open different ports and behave slightly differently across these modes. For more information, see the [stack documentation](https://github.com/appsody/stacks/tree/master/incubator).
```
Comment (mhamwala) [inline content/docs/using-appsody/local-development.md]: possible reword?:
..see the appropriate [stack readme]

File: content/docs/using-appsody/local-development.md [hunk 8]
```diff
@@ -3,26 +3,31 @@ path: /docs/using-appsody/local-development
...
+- `appsody run` starts the development container in normal *run* mode. This mode most closely resembles your application running in production. Appsody watches your local project directory for file changes and hot swaps the application to reflect code changes as you develop.
 
-## appsody debug
-The `appsody debug` command starts the development container with a debugger enabled. Typically, the stack provides a debug port which connects to your IDE. Then you can set breakpoints and step through code as it runs in the container. 
+- `appsody debug` starts the development container with a debugger enabled. Typically, the stack provides a debug port that connects to your IDE. You can then set breakpoints and step through your code as it runs in the container.
 
-## appsody test
-The `appsody test` command runs the test suite for your application in the container. See the stack's documentation on how to define tests for the application.
+- `appsody test` runs the test suite for your application in the container. To define tests for the application, see the [stack documentation](https://github.com/appsody/stacks/tree/master/incubator).
```
Comment (mhamwala) [inline content/docs/using-appsody/local-development.md]: same here:
..see the appropriate [stack readme]

File: content/docs/using-appsody/building-and-deploying.md (hunk 1 above)
Comment (SueChaplain) [reply content/docs/using-appsody/building-and-deploying.md]: No I think downloaded and run for you is correct in this instance. Downloaded and ran would not be grammatically correct.

File: content/docs/using-appsody/building-and-deploying.md (hunk 3 above)
Comment (SueChaplain) [reply content/docs/using-appsody/building-and-deploying.md]: Yes, that's better

File: content/docs/using-appsody/building-and-deploying.md (hunk 4 above)
Comment (SueChaplain) [reply content/docs/using-appsody/building-and-deploying.md]: yes better

File: content/docs/using-appsody/initializing-project.md (hunk 6 above)
Comment (SueChaplain) [reply content/docs/using-appsody/initializing-project.md]: Agree it is good to refer to the readme files. Modified to

>review the [documentation](https://github.com/appsody/stacks/tree/master/incubator) for the appropriate stack.

as we are not linking directly to a stack readme.

File: content/docs/using-appsody/local-development.md (hunk 7 above)
Comment (SueChaplain) [reply content/docs/using-appsody/local-development.md]: modified as above

File: content/docs/using-appsody/local-development.md (hunk 8 above)
Comment (SueChaplain) [reply content/docs/using-appsody/local-development.md]: modified as above

**Comment by SueChaplain** [comment]
@mhamwala - Hi Musa. Thanks for the review. I've force pushed some changes but I've no more time for anything before I go on holiday. If you see further updates, suggest you merge and then fix?

File: content/docs/using-appsody/initializing-project.md (hunk 6 above)
Comment (mhamwala) [reply content/docs/using-appsody/initializing-project.md]: sounds good!

**Pull request merged**